source/development/coredump.rst

```diff
@@ -117,3 +117,46 @@ On macOS, most applications have core dump generation disabled by default. Howev
       # sysctl -w kern.corefile=/cores/core.%P
 
 Enabling core dump generation might consume significant disk space, so use it judiciously. Moreover, not all processes on macOS support or behave consistently with core dump generation.
+
+Windows endpoints
+-----------------
+    .. note:: Remember to be cautious when modifying the Windows Registry. Incorrect changes can cause system instability or other issues. Always backup the registry or create a system restore point before making changes.
```

Comment:
We can remove this note and add a step to instruct about backing up, preferably linking to official Windows documentation on how to do this.